src/types/timestamp.rs

```diff
@@ -572,6 +572,56 @@ impl Timestamp {
     pub fn precision(&self) -> Precision {
         self.precision
     }
+
+    /// Returns the year that has been specified in the [Timestamp].
+    pub fn year(&self) -> i32 {
+        self.date_time.year()
+    }
+
+    /// Returns the month that has been specified in the [Timestamp].
+    pub fn month(&self) -> u32 {
+        self.date_time.month()
+    }
+
+    /// Returns the day that has been specified in the [Timestamp].
+    pub fn day(&self) -> u32 {
+        self.date_time.day()
+    }
+
+    /// Returns the hour(s) that has been specified in the [Timestamp].
+    pub fn hour(&self) -> u32 {
+        // verify if the timestamp has an offset
+        if let Some(offset) = self.offset {
+            // `NativeDateTime#hours()` returns hours normalized as per UTC
+            // for local time we need to +/- the difference
+            let local_date_time = DateTime::<FixedOffset>::from_utc(self.date_time, offset);
+            return local_date_time.hour();
+        }
+        self.date_time.hour()
+    }
+
+    /// Returns the minute(s) that has been specified in the [Timestamp].
+    pub fn minute(&self) -> u32 {
+        // verify if the timestamp has an offset
+        if let Some(offset) = self.offset {
+            // `NativeDateTime#hours()` returns minutes normalized as per UTC
+            // for local time we need to +/- the difference
+            let local_date_time = DateTime::<FixedOffset>::from_utc(self.date_time, offset);
+            return local_date_time.minute();
+        }
+        self.date_time.minute()
+    }
+
+    /// Returns the second(s) that has been specified in the [Timestamp].
+    pub fn second(&self) -> u32 {
+        self.date_time.second()
+    }
+
+    /// Return a UTC timestamp for this [Timestamp]
+    pub fn to_utc(&self) -> Timestamp {
+        let utc_date_time = DateTime::<Utc>::from_utc(self.date_time, Utc);
+        utc_date_time.naive_utc().into()
```

Comment:
Is this different from doing:
```suggestion
        self.date_time.into()
```
?

Reply:
Yes, that is true both are same.